### PR TITLE
Add back commands template

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: src/generate/renders/tests/ScenarioTool.js [hunk 1]
```diff
@@ -0,0 +1,978 @@
+"use strict";
```
Comment (changlong-liu) [inline src/generate/renders/tests/ScenarioTool.js:1]: is this file miss commited? seems ScenarioTool.ts is enough

Comment (qiaozha) [reply src/generate/renders/tests/ScenarioTool.js:1]: oh yes. let me delete it

**Review by changlong-liu — APPROVED** [review]

File: src/python/install.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    if venv_prexists:
+        env_builder = venv.EnvBuilder(with_pip=True)
+        venv_context = env_builder.ensure_directories(venv_path)
+    else:
+        env_builder = ExtendedEnvBuilder(with_pip=True)
+        env_builder.create(venv_path)
+        venv_context = env_builder.context
+        python_run(venv_context, "pip", ["install", "-U", "pip"])
+        python_run(venv_context, "pip", ["install", "-r", "src/python/requirements.txt"])
```
[the remark this answers is not in the document]
Comment (qiaozha) [reply src/python/install.py]: added.

File: src/generate/renders/tests/ScenarioTool.js (hunk 1 above)
Comment (qiaozha) [reply src/generate/renders/tests/ScenarioTool.js]: removed

**Review by lirenhe — APPROVED** [review]

**Pull request merged**